WaterLog_Backend/Procedures.cs

```diff
@@ -368,8 +368,12 @@ public async Task<double> CalculateTotalUsageLitres(SegmentLeaksEntry leak)
                 {
                     totalUsageForPeriod += (item.FlowIn / 60);
                 }
-
-                return totalUsageForPeriod;
+                var hours = (leak.LatestTimeStamp - leak.OriginalTimeStamp).TotalHours;
```

Comment:
Use `Math.Max` so you don't have to assign twice.

Reply:
Using Math.Max as suggested